frontend/src/v5/ui/components/legal/legalTemplate/legalTemplate.component.tsx

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Instead of legalTemplate can we call it `<LegalLayout />` ? Just to keep it consistent with dashboardlayout (I think both serve the same putpose in their respective contexts)

Reply:
renamed